### ISSUE TITLE
Docs improvement

### ISSUE THREAD

**Issue by muco-rolle** [adonisjs/v5-docs#139]
In Authorization - [ Using different User models
](https://docs.adonisjs.com/guides/authorization)

The docs mention to Check out the following video as an example of the same. but there's no video after
![docs-error](https://user-images.githubusercontent.com/43722815/160166172-258d0817-79d9-4004-b0fa-95fe1f5f7d01.png)

**Comment by Barbapapazes** [comment]
Yes and nothing [is in the source code of this page!](https://github.com/adonisjs/docs.adonisjs.com/blob/develop/content/guides/digging-deeper/authorization.md?plain=1#L107)

**Comment by nazarepiedady** [comment]
@thetutlage we need to update the readme file of this repository to explain clearly how to serve this repository in production.
